conf/machine/accton-csp7551.conf

```diff
@@ -18,6 +18,9 @@ require conf/machine/include/x86/x86-base.inc
 # https://github.com/bisdn/meta-open-network-linux/blob/master/conf/machine/include/onl.inc
 require conf/machine/include/onl.inc
 
+# Don't install onlpd/libonlp*
+BISDN_SWITCH_IMAGE_EXTRA_INSTALL:remove = "onl"
```

Comment:
I tried to make the platform look as close to the other platforms as possible. Are we going to add new platforms to meta-switch instead of meta-open-network-linux when vendors only support SONiC?